language/rescue_spec.rb

```diff
@@ -290,4 +290,11 @@ def rescuer.===(exception)
       :expected
     end.should == :expected
   end
+
+  ruby_version_is "2.4" do
+    it "parses 'rescue' in method arguments" do
+      two = eval %{1.+ (raise("Error") rescue 1)}
```

Comment:
Could you use single quotes `'` here? I think it would be slightly more readable.

Reply:
Of course!